frontend/app/src/components/address-book-manager/AddressBookManagement.vue

```diff
@@ -120,84 +120,64 @@ setSubmitFunc(save);
-              input-class=""
-              hide-details
-              prepend-inner-icon="mdi-magnify"
-              :label="t('common.actions.filter')"
-              outlined
-              dense
-              clearable
-              @input="onSearchTermChange($event)"
-            >
-              <template v-if="isTimeoutPending" #append>
-                <VProgressCircular
-                  indeterminate
-                  color="primary"
-                  width="2"
-                  size="24"
-                />
-              </template>
-            </VTextField>
-          </VCol>
-        </VRow>
-      </template>
-
-      <VRow align="center">
-        <VCol cols="auto">
-          <VTabs v-model="tab">
-            <VTab v-for="loc in locations" :key="loc">
-              {{ loc }}
-            </VTab>
-          </VTabs>
-        </VCol>
-        <VCol class="pl-0">
-          <EthNamesHint with-header />
-        </VCol>
-        <VCol />
-        <VCol cols="auto">
-          <VBtn
-            class="mr-2"
-            small
-            depressed
-            fab
-            color="primary"
-            @click="openForm()"
-          >
-            <VIcon>mdi-plus</VIcon>
-          </VBtn>
-        </VCol>
-      </VRow>
+  <TablePageLayout class="p-4">
+    <template #title>
+      {{ t('navigation_menu.manage_address_book') }}
+    </template>
+    <template #buttons>
+      <RuiButton color="primary" @click="openForm()">
+        <template #prepend>
+          <RuiIcon name="add-line" />
+        </template>
+        {{ t('address_book.dialog.add_title') }}
+      </RuiButton>
+    </template>
+
+    <RuiCard>
+      <div class="flex flex-row flex-wrap items-center justify-end gap-2">
+        <ChainSelect
+          evm-only
+          :model-value="selectedChain"
+          hide-details
+          class="flex-1 max-w-full md:max-w-[22rem]"
+          dense
+          @update:model-value="selectedChain = $event"
+        />
+        <RuiTextField
+          :value="pendingSearch"
+          hide-details
+          prepend-icon
+          class="flex-1 max-w-full md:max-w-[22rem]"
+          dense
+          :label="t('common.actions.filter')"
+          variant="outlined"
+          @input="onSearchTermChange($event)"
+        >
+          <template v-if="isTimeoutPending" #append>
+            <div class="flex flex-row items-center">
+              <RuiProgress
+                variant="indeterminate"
+                circular
+                size="20"
+                thickness="3"
+                color="primary"
+              />
+            </div>
+          </template>
+        </RuiTextField>
+      </div>
+
+      <div class="flex flex-row gap-2">
+        <RuiTabs v-model="tab" color="primary">
+          <RuiTab v-for="loc in locations" :key="loc" class="capitalize">
+            {{ loc }}
+          </RuiTab>
+        </RuiTabs>
+        <EthNamesHint with-header />
```

Comment:
the hint looks like egg 🤣
<img width="672" alt="image" src="https://github.com/rotki/rotki/assets/26648140/abc46d5f-da85-41a9-8502-a6d6f05ee329">